### PR TITLE
Update build-integrate-assets.md

### PR DESCRIPTION
**Pull request by bee344**
Added excerpt for XCM Monitoring

### CODE REVIEW

File: docs/build/build-integrate-assets.md [hunk 1]
```diff
@@ -135,3 +135,27 @@ offline environments. It comes with asset-specific functions to use on the Asset
...
+
+Thanks to XCM and a growing number of parachains, {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+can exist across several blockchains, which means the providers need to monitor cross-chain transfers 
+on top of local transfers and its corresponding `balances.transfer` events.
+
+Currently {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} can be sent and received in the Relay 
+Chain either with a [Teleport](https://wiki.polkadot.network/docs/learn-teleport) from 
+[system parachains](https://wiki.polkadot.network/docs/learn-system-chains) or with a 
+[Reserve Backed Transfer](https://wiki.polkadot.network/docs/learn-xcm-pallet#transfer-reserve-vs-teleport) 
+from any other parachain. In both cases, the event emitted when processing the transfer is the `balances.desposited`
```
Comment (Imod7) [inline docs/build/build-integrate-assets.md]:
```suggestion
from any other parachain. In both cases, the event emitted when processing the transfer is the `balances.deposit` 
```

File: docs/build/build-integrate-assets.md [hunk 2]
```diff
@@ -135,3 +135,27 @@ offline environments. It comes with asset-specific functions to use on the Asset
...
+Thanks to XCM and a growing number of parachains, {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+can exist across several blockchains, which means the providers need to monitor cross-chain transfers 
+on top of local transfers and its corresponding `balances.transfer` events.
+
+Currently {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} can be sent and received in the Relay 
+Chain either with a [Teleport](https://wiki.polkadot.network/docs/learn-teleport) from 
+[system parachains](https://wiki.polkadot.network/docs/learn-system-chains) or with a 
+[Reserve Backed Transfer](https://wiki.polkadot.network/docs/learn-xcm-pallet#transfer-reserve-vs-teleport) 
+from any other parachain. In both cases, the event emitted when processing the transfer is the `balances.desposited` 
+event. Hence providers should listent for these events pointing to an addresss in their system. For this,
```
Comment (Imod7) [inline docs/build/build-integrate-assets.md]:
```suggestion
event. Hence providers should listen for these events pointing to an addresss in their system. For this, 
```

File: docs/build/build-integrate-assets.md [hunk 3]
```diff
@@ -135,3 +135,27 @@ offline environments. It comes with asset-specific functions to use on the Asset
...
+[system parachains](https://wiki.polkadot.network/docs/learn-system-chains) or with a 
+[Reserve Backed Transfer](https://wiki.polkadot.network/docs/learn-xcm-pallet#transfer-reserve-vs-teleport) 
+from any other parachain. In both cases, the event emitted when processing the transfer is the `balances.desposited` 
+event. Hence providers should listent for these events pointing to an addresss in their system. For this, 
+the service provider will need to query every new block that is created, loop through the events array, 
+filter for any `balances.deposit` event and apply the appropriate business logic.
+
+#### Tracking back XCM information
+
+What's previously stated will be sufficient to ensure {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
```
Comment (Imod7) [inline docs/build/build-integrate-assets.md]:
```suggestion
What has been mentioned earlier should be sufficient to confirm that {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
```

File: docs/build/build-integrate-assets.md [hunk 4]
```diff
@@ -135,3 +135,27 @@ offline environments. It comes with asset-specific functions to use on the Asset
 constructing parachain transactions, you can use `txwrapper-polkadot` exactly as on the Relay Chain,
 but construct transactions with the appropriate parachain metadata like genesis hash, spec version,
 and type registry.
+
+### XCM Transfer Monitoring
```
Comment (Imod7) [inline docs/build/build-integrate-assets.md]:
```suggestion
### Monitoring of XCM deposits
```

**Review by filippoweb3 — APPROVED** [review]
lgtm, thank for the addition and the review. I just polished the grammar.

**Pull request merged**